sonar-project.properties

```diff
@@ -14,7 +14,7 @@ sonar.cfamily.build-wrapper-output=build-wrapper-output-dir
 sonar.cfamily.gcov.reportsPath=.
 sonar.coverageReportPaths=cov.xml
 sonar.python.coverage.reportPaths=jormungandr_serpy_integration_coverage.xml,jormungandr_coverage_serpy.xml
-sonar.coverage.exclusions=source/jormungandr/tests/**,source/tyr/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/navitiacommon/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**
+sonar.coverage.exclusions=sonar.coverage.exclusions=source/jormungandr/tests/**,source/tyr/tests/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/navitiacommon/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**
```

Comment:
Are you sure about that? `sonar.coverage.exclusions=sonar.coverage.exclusions=...` ?

Reply:
ohhh nice thank you :)